requirements.txt

```diff
@@ -10,4 +10,3 @@ gunicorn==20.0.4
 psycopg2-binary==2.8.6
 python-dotenv==0.18.0
 six==1.11.0
-pre-commit
```

Comment:
Why pre-commit has been removed?

Reply:
Sorry, it was a mistake from my side. Will change it.